### PR TITLE
Logs rate limit control documentation

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Describe how to set a daily quota on log indexes.

### Motivation
The feature was released to everybody

### Preview link

https://docs-staging.datadoghq.com/nils/log-rate-limit-control/logs/indexes/#pagetitle

### CODE REVIEW

File: content/en/logs/indexes/_index.md [hunk 1]
```diff
@@ -68,6 +68,16 @@ Enable or disable them in one click on the Pipeline page:
...
 
+### Set daily quota
+
+You can set a daily quota to hard-limit the number of logs that are stored within an Index per day. This quota is applied for all logs that should have been stored (i.e after exclusion filters are applied). Once the daily quota is reached, logs are not indexed anymore and go to your Archives if those are set.
+
+Update or remove this quota at any time when editing the Index: 
+
+{{< img src="logs/indexes/index_quota.png" alt="index details" responsive="true" style="width:70%;">}}
+
+**Note**: Indexes quota resets automatically at 12:00pm UTC (2:00pm CET, 8:00am EDT, 5:00am PDT).
```
Comment (l0k0ms) [inline content/en/logs/indexes/_index.md]:
```suggestion
**Note**: Indexes quota resets automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
```

**Comment by NBParis** [comment]
Thanks for the update @l0k0ms

File: content/en/logs/indexes/_index.md [hunk 2]
```diff
@@ -68,6 +68,16 @@ Enable or disable them in one click on the Pipeline page:
 
 {{< img src="logs/indexes/enable_index_filters.png" alt="enable index filters" responsive="true" style="width:80%;">}}
 
+### Set daily quota
+
+You can set a daily quota to hard-limit the number of logs that are stored within an Index per day. This quota is applied for all logs that should have been stored (i.e after exclusion filters are applied). Once the daily quota is reached, logs are not indexed anymore and go to your Archives if those are set.
```
Comment (l0k0ms) [inline content/en/logs/indexes/_index.md]:
```suggestion
You can set a daily quota to hard-limit the number of logs that are stored within an Index per day. This quota is applied for all logs that should have been stored (i.e after exclusion filters are applied). Once the daily quota is reached, logs are no longer indexed and go into your Archives if those are set.
```

**Comment by NBParis** [comment]
@l0k0ms did a small update to highlight the feature that are still available when limits are hit.

**Pull request merged**